src/components/export-modal/export-modal.less

```diff
@@ -1,6 +1,6 @@
 @import (reference) "~less/compass/_theme.less";
 
-.export-to-lang-modal-checkbox {
+.export-to-lang-modal-checkbox-& {
```

Comment:
Is this right?

Reply:
@aherlihy i haven't seen this syntax before! what is this meant to do?

Reply:
I think it's for applying it to any style with that ending, but tbh I haven't used for a long time so I'm not sure it applies here.

Reply:
if it works the way you expect it, then go for it!